contrib/valgrind.supp

```diff
@@ -277,3 +277,13 @@
    Memcheck:Cond
    fun:gdr_unmap
 }
+{
+    is_cuda_supported_unint_access
+    Memcheck:Param
+    ioctl(generic)
+    fun:ioctl
+    ...
+    fun:cuDeviceTotalMem_v2
```

Comment:
why did it start to fail?
(recent change on GPU nodes was to load nv_peer_mem.ko)

Reply:
I don't know. I just found that it started to fail... may be internal updates in valgrind or CUDA...